sys-apps/grep/grep-3.5.ebuild

```diff
@@ -13,9 +13,12 @@ SRC_URI="mirror://gnu/${PN}/${P}.tar.xz
 LICENSE="GPL-3+"
 SLOT="0"
 KEYWORDS="~alpha amd64 arm arm64 hppa ~ia64 ~m68k ~mips ppc ppc64 ~riscv ~s390 ~sparc x86 ~x64-cygwin ~amd64-linux ~x86-linux ~ppc-macos ~x64-macos ~sparc-solaris ~sparc64-solaris ~x64-solaris ~x86-solaris"
-IUSE="nls pcre static"
+IUSE="nls libsigsegv pcre static"
 
-LIB_DEPEND="pcre? ( >=dev-libs/libpcre-7.8-r1[static-libs(+)] )"
+REQUIRED_USE="static? ( !libsigsegv )"
```

Comment:
You can add `static-libs` USE flag to `dev-libs/libsigsegv`.
Otherwise you would need ` sys-apps/grep static` in SPARC's `package.use.mask`.

Reply:
Thanks, I'll do that.